Requirements.md

```diff
@@ -0,0 +1,36 @@
+# book-club
+Projects idea is to create an repo for books, which registered users can like, rate and review.
```

Comment:
The idea of project is to create a repo for books, which registered users can like, rate and review.